source/docs/appendices/migration-guides.md

```diff
@@ -43,6 +43,19 @@ The goal is to avoid variants collision and to make it explicit when a variant o
 
 These changes are backward compatible. Deprecation warnings will appear if you keep using the old properties.
 
+### Files loading in a Scss file
+
+Until now, files used in a `.scss` file were to be loaded from the public directory or by using long and tidious file paths. This is no longer the case. You can now use relative imports.
+
+For instance, if you have a `theme/components/atoms/Icon/_Icon.scss` file, you will be able to import the font file directly:
+
+```diff
+-src: url('../theme/components/atoms/Icon/font-awesome-4.7.0/fonts/fontawesome-webfont.woff2') format('woff2'),
```

Comment:
Shouldn’t it be `'~theme/………'` here?

Reply:
It didn't know that `~theme/…` worked previously.

However it still works now, so no need to mention it here.